usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -62,14 +63,26 @@ def test_sles_hardened(host, get_release_value, is_sles_sap, is_sle_micro):
         )
     )
 
+    scap_report = os.environ.get("SCAP_REPORT", "")
+    if scap_report:
+        if os.path.exists(scap_report) or not scap_report.endswith(".html"):
+            pytest.skip("Invalid filename for SCAP_REPORT: {}".format(scap_report))
```

Comment:
DO we really want to skip the whole test just because the user gave us an unacceptable value for the html report? Shouldn't we make a best effort here and still run the test without the report? At least if the test runs we get a pass/fail evaluation.

Reply:
> DO we really want to skip the whole test just because the user gave us an unacceptable value for the html report? Shouldn't we make a best effort here and still run the test without the report? At least if the test runs we get a pass/fail evaluation.

1. From a security perspective it doesn't make sense to continue the test if the input is potentially malicious.
2. In the event that `SCAP_REPORT` was only wrongly set, the test should also be skipped so we notice its absence because of this reason.

Reply:
I'd lean towards the test still running with report disabled if bad value is passed in. But I assume we are running tests on a test instance. If something goes wrong it's ephemeral and can be replaced. That being said avoiding any overwrites adds safety (in case for some reason somebody runs this on a production instance).

Reply:
> I'd lean towards the test still running with report disabled if bad value is passed in. But I assume we are running tests on a test instance. If something goes wrong it's ephemeral and can be replaced. That being said avoiding any overwrites adds safety (in case for some reason somebody runs this on a production instance) with minimal effort.

And this PR was already tested in openQA. It works as intended.

Reply:
I agree with @smarlowucf we should do a best effort implementation. If the value from the environment is bad we ignore it an run the test.